server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1240,13 +1243,12 @@ protected void runInContext() {
             try {
                 final List<DomainRouterVO> routers = _routerDao.listByStateAndManagementServer(VirtualMachine.State.Running, mgmtSrvrId);
                 s_logger.info("Found " + routers.size() + " running routers. Fetching, analysing and updating DB for the health checks.");
+                if (!RouterHealthChecksEnabled.value()) {
+                    s_logger.debug("Skipping fetching of router health check results as router.health.checks.enabled is disabled");
+                    return;
+                }
```

Comment:
-\o/- , not a useful log message to those that see it , i think.

Reply:
I can remove the debug log if you think it's irrelevant? The other paths are lgged above the if and subsequent subroutines. Thoughts?